Gemfile.lock

```diff
@@ -61,19 +63,18 @@ GEM
       nokogiri (>= 1.6)
     rails-html-sanitizer (1.3.0)
       loofah (~> 2.3)
-    railties (6.0.3.3)
-      actionpack (= 6.0.3.3)
-      activesupport (= 6.0.3.3)
+    railties (6.1.3)
+      actionpack (= 6.1.3)
+      activesupport (= 6.1.3)
       method_source
       rake (>= 0.8.7)
-      thor (>= 0.20.3, < 2.0)
-    rake (13.0.1)
-    thor (1.0.1)
-    thread_safe (0.3.6)
-    tzinfo (1.2.7)
-      thread_safe (~> 0.1)
-    zeitwerk (2.4.0)
-    zxcvbn-ruby (1.1.0)
+      thor (~> 1.0)
+    rake (13.0.3)
+    thor (1.1.0)
+    tzinfo (2.0.4)
```

Comment:
### [tzinfo](https://tzinfo.github.io)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 1.2.7 → 2.0.4

[_[change-log](https://github.com/tzinfo/tzinfo/blob/master/CHANGES.md), [source-code](https://github.com/tzinfo/tzinfo/tree/v2.0.4)_]





<details>
<summary>Commits</summary>

A change of **491** commits. See the full changes on [the compare page](https://github.com/tzinfo/tzinfo/compare/v1.2.7...v2.0.4).

These are the first 10 commits:
- (e45e946) [Delete symlinks created in temporary directories.](https://github.com/tzinfo/tzinfo/commit/e45e9462dd317fb9bc356544f20640733944fa4f)
- (70a3726) [Fix the zoneinfo file permission denied test on Windows.](https://github.com/tzinfo/tzinfo/commit/70a3726d2eecdd2b8724a86b28dfde090da4fb8a)
- (efd74e4) [Add the AppVeyor build status badge.](https://github.com/tzinfo/tzinfo/commit/efd74e4396f0321765765ea536975297aa6b49a5)
- (5a8da93) [Wrap exception handling around just the call to read.](https://github.com/tzinfo/tzinfo/commit/5a8da9338c86fa76af1bfd3d9d6268877bdcbc74)
- (471cf65) [Remove the Timezone.us_zones and Timezone.us_zone_identifier…](https://github.com/tzinfo/tzinfo/commit/471cf659ac9e8c6486916e82b35ca0b291d66602)
- (10a910a) [Use 'must be specified' as the message for `unless param` ex…](https://github.com/tzinfo/tzinfo/commit/10a910ac0d8ce44ab904586200d15e219e831e51)
- (8a92e3c) [Use File.join instead of hardcoded directory separators.](https://github.com/tzinfo/tzinfo/commit/8a92e3c66fe8f4f59c89db989957fc53d5031b2e)
- (0a09419) [Fix a typo in a comment.](https://github.com/tzinfo/tzinfo/commit/0a094195c8023d5d8aa08560c1c647508e0e87a5)
- (0e5727c) [Update documentation using YARD syntax.](https://github.com/tzinfo/tzinfo/commit/0e5727cacf486056c842e49a984937798d1da627)
- (845ddcd) [Add Ruby 2.5.0, update to Ruby 2.2.9, 2.3.6, 2.4.3 and JRuby…](https://github.com/tzinfo/tzinfo/commit/845ddcdccfe99c68441ddd276eb24d178b30e9cb)


</details>